### ISSUE TITLE
TIFF writer not available via maven artifacts

### ISSUE THREAD

**Issue by oliverwelter** [haraldk/TwelveMonkeys#142]
Hi,
I'm was trying to use the tiff plugin as maven artifact in a gradle project.
I added

compile 'com.twelvemonkeys.imageio:imageio-tiff:3.1.0'`

into the gradle.build file. Afterwards a tiff reader but no writer was available.
I wrote a small unit test showing supported mime types
```java
@Test
public void testFormats() {
  ImageIO.scanForPlugins();
  System.out.println("Available readers:");
  System.out.println(Arrays.toString(ImageIO.getReaderMIMETypes()));
  System.out.println("Available writers:");
  System.out.println(Arrays.toString(ImageIO.getWriterMIMETypes()));
}
```
Before adding the imageio-tiff dependency I got the following output
> Available readers:
> [image/png, image/jpeg, image/x-png, image/tiff, image/vnd.wap.wbmp, image/bmp, image/gif, image/x-tiff]
> Available writers:
> [image/png, image/jpeg, image/x-png, image/vnd.wap.wbmp, image/gif, image/bmp]`

Afterwards:
> Available readers:
> [image/png, image/jpeg, image/x-png, image/tiff, image/vnd.wap.wbmp, image/gif, image/bmp, image/x-tiff]
> Available writers:
> [image/jpeg, image/png, image/x-png, image/vnd.wap.wbmp, image/bmp, image/gif]

Do you have any suggestion what could be the reason that the writer plugin isn't found?
The gradle dependencies are showing the correct jar and within it I can see TIFFImageWriter.class and TIFFImageWriterSpi.class .

Best Regards,
Oliver

**Comment by oliverwelter** [comment]
I was able to work around the problem by registering the plugin manually:
```java
static {
    IIORegistry registry = IIORegistry.getDefaultInstance();
    registry
        .registerServiceProvider(new com.twelvemonkeys.imageio.plugins.tiff.TIFFImageWriterSpi());
}
```

**Comment by haraldk** [comment]
Hi Oliver,

Thanks for reporting! 

This is actually quite embarrassing.. I forgot to check in the resource file that is used to register the writer! :-O

Anyway, good catch! I'll fix ASAP. For now, your workaround is okay, I guess. I hope to make a 3.1.1 bugfix release containing this fix soon.


Regards,

-- 
Harald K

**Comment by haraldk** [comment]
Fixed in 8052fbc5b88669f4a475d9ef2c7e8274e114fdf7.

-- 
Harald K

**Comment by haraldk** [comment]
Released in [3.1.1](https://github.com/haraldk/TwelveMonkeys/releases/tag/twelvemonkeys-3.1.1) (bugfix release).
